biocypher_metta/adapters/uberon_adapter.py

```diff
@@ -0,0 +1,20 @@
+from biocypher_metta.adapters.ontologies_adapter import OntologyAdapter
+
+class UberonAdapter(OntologyAdapter):
+    ONTOLOGIES = {
+            'uberon': 'http://purl.obolibrary.org/obo/uberon.owl'
+        }
+
+    def __init__(self, write_properties, add_provenance, ontology, type, label='uberon', dry_run=False):
+        super(UberonAdapter, self).__init__(write_properties, add_provenance, ontology, type, label, dry_run)
+
+    
+    def get_ontology_source(self, ontology):
+        """
+        Returns the source and source URL for UBERON ontology.
+        """
+        if ontology == 'uberon':
```

Comment:
This is the UBERON class, so why do you need an if-else statement. You should just have the return statement